build.gradle

```diff
@@ -89,7 +89,7 @@ allprojects {
                 mavenBom "org.junit:junit-bom:${junitVersion}"
             }
             dependencies {
-                dependency "org.yaml:snakeyaml:${snakeYamlVersionOverride}"
+//                dependency "org.yaml:snakeyaml:${snakeYamlVersionOverride}"
```

Comment:
The snakeyaml dependency version from the gradle.properties, should be used here

Reply:
That override broke some things because it was incompatible with, I believe: liquibase. But I'll check.

Reply:
@Klaas-Ritense Added it again.